src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss

```diff
@@ -1,13 +1,13 @@
-@import "../../../../styles/states.scss";
-@import "../../../../styles/global-css-settings.scss";
+@import "../../../../styles/states";
+@import "../../../../styles/global-css-settings";
 
 .monday-style-color-item-wrapper {
   --border-radius-multiplier: 1;
 
   &.circle {
     --border-radius-multiplier: 99999;
   }
-
+  /* stylelint-disable scss/operator-no-unspaced */
   @mixin absolute-border($border-width, $border-color-var-name, $inset: -$border-width, $border-radius: 7px) {
```

Comment:
why did you need the disable - was there a problem with `$inset: -$border-width`? 
if so, should we fix instead of disabling?

Reply:
@laviomri will fix it after merge,
this branch should have no "potential" change